etcdmain/config.go

```diff
@@ -321,7 +321,7 @@ func (cfg *config) configFromCmdLine() error {
 		}
 	}
 
-	err := flags.SetFlagsFromEnv("ETCD", cfg.cf.flagSet)
+	err := flags.SetFlagsFromEnv(cfg.ec.GetLogger(), "ETCD", cfg.cf.flagSet)
```

Comment:
Note: At this point, I don't think user-configured zap logger is properly setup yet. Should we use zap.NewProduction() or maybe a different logger?

Reply:
`zap.NewProduction` should be fine.